### ISSUE TITLE
Multiselect Fields needs checkboxes instead of Radio Buttons

### ISSUE THREAD

**Issue by pbatroff** [systopia/civiremote#1]
Wehn using Multi Select Fields from the Remoteevent Form Generator, Multi-Select Fields < 10 options are shown with radio buttons, thus making it not possible to select more than one value from this. 

@jensschuppe implemented a quick fix in https://github.com/systopia/civiremote/compare/fix/multiSelectCheckboxes, but after this empty values are transmitted as well, making the API call fail (0 isn't a valid option).

Example call:
```json
{
    "event_id": "3",
    "remote_contact_id": null,
    "profile": "fb-3",
    "first_name": "ObiWan",
    "middle_name": "Todd",
    "last_name": "Kenobi",
    "custom_18": {
        "2": "2",
        "1": 0,
        "3": 0,
        "4": 0,
        "5": 0,
        "6": 0,
        "7": 0,
        "8": 0,
        "9": 0
    },
    "custom_19": "1",
    "token": null,
    "context": "create",
    "options": [],
    "entity": "RemoteParticipant",
    "action": "validate"
}
```

**Comment by pbatroff** [comment]
Error Message when submitting this:

`Couldn't find or create contact: '0' is not a valid option for field custom_18`

**Comment by pbatroff** [comment]
Error Message when submitting this:

`Couldn't find or create contact: '0' is not a valid option for field custom_18`

**Comment by jensschuppe** [comment]
This should have been fixed with 1d7c13a92d66e12e2e86921da0e733603ed45edb.
